### PR TITLE
Remake athlete/roster scraper

### LINKED ISSUE #19: Mixups in athlete parameters
Name field does not correspond to the appropriate value

```
{
    "0": "Fr.",
    "Hometown": "Woodway, Wash. (Edmonds-Woodway)",
    "height": "GK",
    "name": "1",
    "position": "Kiera Towell",
    "year": "5-10"
}
```

### PR DESCRIPTION
**Pull request by adelq**
Fixes #19, #21. Test results (compared to master) should be self-explanatory

### CODE REVIEW

File: pennathletics/events.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from munch import munchify
+import requests
+import arrow
+
+EVENTS_URL = ("http://www.pennathletics.com/v3/events?page=1&limit=40"
```
Comment (armic) [inline pennathletics/events.py]: Maybe better to include the base url here?

**Review by armic — APPROVED** [review]
👍

File: pennathletics/events.py (hunk 1 above)
Comment (adelq) [reply pennathletics/events.py]: Good call. Also fixing the failing doctest

**Comment by adelq** [comment]
@ajmichaels

**Comment by armic** [comment]
👍

**Pull request merged**